src/ecs/resources/input/input_handler.rs

```diff
@@ -260,7 +285,7 @@ impl InputHandler {
     }
 
     /// Returns the value of an axis by the string id, if the id doesn't exist this returns None.
-    pub fn axis_value<T: AsRef<str>>(&self, id: T) -> Option<f32> {
+    pub fn axis_value<T: AsRef<str>>(&self, id: T) -> Option<f64> {
```

Comment:
I think the same function will be used for joysticks axes. But yeah, f64 seems unnecessary precise.